src/test/java/com/ays/institution/controller/InstitutionControllerSystemTest.java

```diff
@@ -0,0 +1,95 @@
+package com.ays.institution.controller;
+
+import com.ays.AbstractSystemTest;
+import com.ays.common.model.dto.response.AysResponse;
+import com.ays.common.model.dto.response.AysResponseBuilder;
+import com.ays.common.util.exception.model.AysError;
+import com.ays.institution.model.Institution;
+import com.ays.institution.model.dto.response.InstitutionResponse;
+import com.ays.institution.model.entity.InstitutionBuilder;
+import com.ays.institution.model.mapper.InstitutionToInstitutionResponseMapper;
+import com.ays.institution.service.InstitutionService;
+import com.ays.util.AysMockMvcRequestBuilders;
+import com.ays.util.AysMockResultMatchersBuilders;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+
+import java.util.Arrays;
+import java.util.List;
+
+class InstitutionControllerSystemTest extends AbstractSystemTest {
+
+    @MockBean
+    private InstitutionService institutionService;
```

Comment:
End-To-End (System) testlerinde servisleri mocklamıyoruz. Servisler gerçekten de çalışmalı.